### PR TITLE
wire up bridge button

### PR DESCRIPTION
**Pull request by sentilesdal**

### CODE REVIEW

File: apps/hyperdrive-trading/src/ui/bridge/BridgeAssetsForm/BridgeAssetsActionButtons.tsx [hunk 1]
```diff
@@ -91,12 +94,7 @@ export function BridgeAssetsActionButtons({
                   quote={quote}
                   balance={chainBalance}
                 />
-                <button
-                  type="button"
-                  className="daisy-btn daisy-btn-circle daisy-btn-primary w-32 disabled:bg-primary disabled:text-base-100 disabled:opacity-30"
-                >
-                  Bridge
-                </button>
+                {BridgeButton()}
```
Comment (ryangoree) [inline apps/hyperdrive-trading/src/ui/bridge/BridgeAssetsForm/BridgeAssetsActionButtons.tsx:97]: What's goin on here?
```suggestion
                <BridgeButton quote={quote} />
```

File: apps/hyperdrive-trading/src/ui/bridge/BridgeAssetsForm/BridgeAssetsActionButtons.tsx [hunk 2]
```diff
@@ -149,6 +147,64 @@ function ApproveButton({
   );
 }
 
+interface BridgeButtonProps {
+  solution: EntityTokenTransferQuote;
```
Comment (ryangoree) [inline apps/hyperdrive-trading/src/ui/bridge/BridgeAssetsForm/BridgeAssetsActionButtons.tsx:151]:
```suggestion
  quote: EntityTokenTransferQuote;
```

File: apps/hyperdrive-trading/src/ui/bridge/BridgeAssetsForm/BridgeAssetsActionButtons.tsx [hunk 3]
```diff
@@ -149,6 +147,64 @@ function ApproveButton({
...
+        <div className="daisy-loading daisy-loading-spinner" />
+      ) : (
+        "Bridge"
+      )}
+    </button>
+  );
+}
+
+function useSendBridgeTransaction(
+  solution: EntityTokenTransferQuote | undefined,
```
Comment (ryangoree) [inline apps/hyperdrive-trading/src/ui/bridge/BridgeAssetsForm/BridgeAssetsActionButtons.tsx:174]:
```suggestion
  quote: EntityTokenTransferQuote | undefined,
```

**Review by ryangoree — APPROVED** [review]

**Comment by DannyDelott** [comment]
Closing in favor of iframe widget from squid router. React component widget is only compatible with NextJS builds :(

**Pull request closed without merging**